example/bayesian-methods/algos.py

```diff
@@ -1,9 +1,10 @@
+from __future__ import print_function
 import mxnet as mx
 import mxnet.ndarray as nd
 import time
 import logging
 from utils import *
-
+from past.builtins import xrange
```

Comment:
past should be included in the __future__  package already. as we would like support 2/3, future package should be part of the dependencies